src/mapbox/extended_nav_control.js

```diff
@@ -21,6 +21,16 @@ export default class ExtendedControl {
     this._compass = this._createButton(compassClass, 'Reset North', () => {
       this._resetNorthAndTilt();
     });
+    this._direction = this._createButton('direction_shortcut icon-corner-up-right', 'direction',
+      () => {
+        Array.prototype.slice.call(
+          document.getElementsByClassName('direction_shortcut')
```

Comment:
I think you don't need an array.
`document.querySelector('direction_shortcut')` will return the one and only component.